scripts/jtl_listener.py

```diff
@@ -170,3 +171,11 @@ def _request_success(self, request_type, name, response_time, response_length, *
     def _request_failure(self, request_type, name, response_time, response_length, exception, **kw):
         self.add_result("false", request_type, name, response_time,
                         response_length, str(exception), **kw)
+
+    def _list_to_csv_row(self, lst):
```

Comment:
is the `io` really needed for this kind of stuff? Its about escaping character(s) right?

Reply:
It's needed as the csv writer class needs a file descriptor to write to. But it shouldn't be a problem, as it is included by default in most of python 3 envs.

About the csv module, I think this is a very basic good practice, it's performant and does not need extra packages to be installed. I'm writing this code because every file including an exception value in any row was just discarded. I have a version escaping chars too, and it is working, but it is just a weaker code. What if floats start using coma for any reason? It's true that this is not that feasible (it's just an example) and it's true too that the environment of the code is close enough to be able to deal with this just escaping some chars, but that way the code is more prone to errors. Using CSV module makes things just safer.

Reply:
I am not worried about availability of the module that much as about the performance penalty this can introduce.